modelscan/cli.py

```diff
@@ -111,12 +132,26 @@ def scan(
             modelscan.scan(pathlibPath)
     else:
         raise click.UsageError("Command line must include a path")
-    ConsoleReport.generate(
-        modelscan.issues,
-        modelscan.errors,
-        modelscan._skipped,
-        show_skipped=show_skipped,
-    )
+
+    report_settings: Dict[str, Any] = {}
+    if format == "custom":
+        reporting_module = settings["reporting"]["module"]  # type: ignore[index]
+        report_settings = settings["reporting"]["settings"]  # type: ignore[index]
+    else:
+        reporting_module = AVAILABLE_REPORTING_MODULES[format]
```

Comment:
Simplify the logic here and just load `settings["reporting"]["module"]` and `settings["reporting"]["settings"]`